addOns/grpc/src/main/java/org/zaproxy/addon/grpc/ExtensionGrpc.java

```diff
@@ -50,4 +87,44 @@ public String getUIName() {
     public String getDescription() {
         return Constant.messages.getString("grpc.desc");
     }
+
+    private static final class ResponseGrpcViewFactory
+            implements HttpPanelManager.HttpPanelViewFactory {
+        public static final String NAME = "ResponseGrpcViewFactory";
+
+        @Override
+        public String getName() {
+            return NAME;
+        }
+
+        @Override
+        public HttpPanelView getNewView() {
+            return new HttpPanelGrpcView(new ResponseBodyStringHttpPanelViewModel());
```

Comment:
I'd have expected a byte model.

Reply:
i thought byte model is hard to edit , let me first add decode/encode algo then we can decide about it.

Reply:
That's how the view will receive the data not how the user will see it.

Reply:
ohh yes, ok i will change it to ResponseBodyByteHttpPanelViewModel and RequestBodyByteHttpPanelViewModel